From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Keeley Hammond <vertedinde@electronjs.org>
Date: Thu, 5 Feb 2026 15:29:44 -0800
Subject: build: restore macos deployment target to 13.0

Partially reverts https://github.com/nodejs/node/commit/8b4022177750530d2c142a5a0349d98fb82f16e2.

Fixes native addons not working on macOS 13.0 - 13.4 (https://github.com/electron/electron/issues/49691).

This patch can be removed when Chromium drops support for macOS 13.

diff --git a/common.gypi b/common.gypi
index becac7d2ae219f56b730fa668f442358d3e849df..456f3d86e49f6cefda6d3a7c93ead63945f74049 100644
--- a/common.gypi
+++ b/common.gypi
@@ -743,7 +743,7 @@
           'GCC_ENABLE_PASCAL_STRINGS': 'NO',        # No -mpascal-strings
           'GCC_STRICT_ALIASING': 'NO',              # -fno-strict-aliasing
           'PREBINDING': 'NO',                       # No -Wl,-prebind
-          'MACOSX_DEPLOYMENT_TARGET': '13.5',       # -mmacosx-version-min=13.5
+          'MACOSX_DEPLOYMENT_TARGET': '13.0',       # -mmacosx-version-min=13.0
           'USE_HEADERMAP': 'NO',
           'WARNING_CFLAGS': [
             '-Wall',
